plugins/modules/rds_cluster_snapshot.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/python
+# Copyright (c) 2014 Ansible Project
+# Copyright (c) 2021 Alina Buzachis (@alinabuzachis)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: rds_cluster_snapshot
+version_added: 4.0.0
+short_description: Manage Amazon RDS snapshots of DB clusters
+description:
+  - Create, modify and delete RDS snapshots of DB clusters.
+options:
+  state:
+    description:
+      - Specify the desired state of the snapshot.
+    default: present
+    choices: [ 'present', 'absent']
+    type: str
+  db_cluster_snapshot_identifier:
+    description:
+      - The identifier of the DB cluster snapshot.
+    required: true
+    aliases:
+      - snapshot_id
+      - id
+      - snapshot_name
+    type: str
+  db_cluster_identifier:
+    description:
+      - The identifier of the DB cluster to create a snapshot for.
+      - Required when I(state=present).
+    aliases:
+      - cluster_id
+      - cluster_name
+    type: str
+  source_db_cluster_snapshot_identifier:
+     description:
+      - The identifier of the DB cluster snapshot to copy.
+      - If the source snapshot is in the same AWS region as the copy, specify the snapshot's identifier.
+      - If the source snapshot is in a different AWS region as the copy, specify the snapshot's ARN.
+     aliases:
+      - source_id
+      - source_snapshot_id
+     type: str
+  source_region:
+    description:
+      - The region that contains the snapshot to be copied.
+    type: str
```

Comment:
This feels a little redundant since cross-region snapshots are supposed to be an ARN (based on source_db_cluster_snapshot_identifier docs)

Reply:
It's an option.  Is there a potential use case where source_region doesn't match the region from the ARN in source_db_cluster_snapshot_identifier ?